contents/docs/integrate/proxy.mdx

```diff
@@ -8,8 +8,8 @@ A reverse proxy allows you to send events to PostHog Cloud using your own domain
 This can have a number advantages, most notably being that events sent to your own domain won't be intercepted by tracker blockers.
 By sending events to your own domain, you'll be able to track more users and more events, without having to host PostHog yourself.
 
-> Note: PostHog Cloud EU requires that the proxy sets the `Host` header to `eu.posthog.com` for requests 
-> sent to https://eu.posthog.com. The instructions below do not account for this. PR's with the requested 
+> Note: PostHog Cloud requires that the proxy sets the `Host` header to `eu.posthog.com` for requests sent to https://eu.posthog.com
+> and `app.posthog.com` for requests sent to https://app.posthog.com. The instructions below do not account for this. PR's with the requested
```

Comment:
This may be coming from a position of technical ignorance (I don't know how much these instructions would need to change), but I feel pretty strongly we should be supplying complete and up-to-date instructions and not just asking users to do it for us. 

Recognise this was in place before this change, so tagging @ivanagas and @pjhul so it can be updated -- especially as we're about to email all US customers and ask them to refer to this page.

Reply:
I think it's worth giving advice for the most common proxy solutions people have. e.g. CloudFlare Proxy CNAME what should they do. I think the answer is don't use CloudFlare. What is the recommended alternative with example config. Then left to the user of other solutions to imitate that setup.